k2/csrc/aux_labels_test.cc

```diff
@@ -33,7 +34,7 @@ class AuxLablesTest : public ::testing::Test {
 TEST_F(AuxLablesTest, AuxLabels1Mapper) {
   {
     // empty arc_map
-    std::vector<int32_t> arc_map;
+    Array1<int32_t *> arc_map;
```

Comment:
Should we define
```cpp
using IntArray1 = Array1<int32_t *>;
```

Reply:
I'd rather use the template directly, but are you sure it should be int32_t*  ?  Seems to me it should be int32_t.

Reply:
I'd prefer to use the template rather than a typedef.
Incidentally, all of this code is going to have to be changed to use the templates defined in the cuda/ directory, like Array1 (it's different), Ragged2, Ragged3.

Reply:
It's `int32_t*`, as we define `Array` to accept `Ptr` (raw pointer or StridedPtr) as its template argument. 
RE Ragged2, yes, of couse we will update those bindings later after cuda interface is stable.

Reply:
Resolved, will not change.